File: src/main/java/de/twometer/amongus/model/PlayerColor.java
package de.twometer.amongus.model;

import de.twometer.neko.render.Color;

public enum PlayerColor {
    Red(fromHex("#FF0000")),
    DarkRed(fromHex("#540000")),
    Green(fromHex("#006836")),
    Lime(fromHex("#00FF00")),
    Blue(fromHex("#0000FF")),
    LightBlue(fromHex("#3287FF")),
    Cyan(fromHex("#00FFFA")),
    Yellow(fromHex("#FFB100")),
    Orange(fromHex("#FF6100")),
    Pink(fromHex("#FF00D1")),
    Purple(fromHex("#6400B7")),
    Brown(fromHex("#612500")),
    White(fromHex("#ffffff")),
    Black(fromHex("#000000")),
    Gray(fromHex("#494949"));


    private final Color value;

    PlayerColor(Color value) {
        this.value = value;
    }

    public Color getValue() {
        return value;
    }

    private static Color fromHex(String hex) {
        java.awt.Color c = java.awt.Color.decode(hex);
        return new Color(c.getRed() / 255.0f, c.getGreen() / 255.0f, c.getBlue() / 255.0f, c.getAlpha() / 255.0f);
    }

}

File: src/main/kotlin/de/twometer/amogus/model/PlayerColor.kt
package de.twometer.amogus.model

import de.twometer.neko.scene.Color

private fun fromHey(hex: String): Color {
    val c = java.awt.Color.decode(hex)
    return Color(c.red / 255.0f, c.green / 255.0f, c.blue / 255.0f, c.alpha / 255.0f)
}

enum class PlayerColor(val color: Color) {
    Red(fromHey("#FF0000")),
    DarkRed(fromHey("#540000")),
    Green(fromHey("#006836")),
    Lime(fromHey("#00FF00")),
    Blue(fromHey("#0000FF")),
    LightBlue(fromHey("#3287FF")),
    Cyan(fromHey("#00FFFA")),
    Yellow(fromHey("#FFB100")),
    Orange(fromHey("#FF6100")),
    Pink(fromHey("#FF00D1")),
    Purple(fromHey("#6400B7")),
    Brown(fromHey("#612500")),
    White(fromHey("#ffffff")),
    Black(fromHey("#000000")),
    Gray(fromHey("#494949"));
}